ControlRoomApplication/ControlRoomApplication/GUI/DiagnosticsForm.cs

```diff
@@ -279,7 +279,16 @@ private void timer1_Tick(object sender, System.EventArgs e)
             _azEncoderDegrees = currAbsOrientation.Azimuth;
             _elEncoderDegrees = currAbsOrientation.Elevation;
             lblAzAbsPos.Text = Math.Round(_azEncoderDegrees, 2).ToString();
-            lblElAbsPos.Text = Math.Round(_elEncoderDegrees, 2).ToString();
+
+            // Check if elevation encoder is timed out, output ERR if so, otherwise output current position
+            if (DatabaseOperations.GetSensorStatusData().elevation_abs_encoder.Equals((SByte)SensorStatusEnum.ALARM))
```

Comment:
We should be checking the Sensor Network Status here directly from the telescope's senor network server.

- [ ] Change check to using sensor network status directly